backend/src/resolvers/conversationPresence.js

```diff
@@ -0,0 +1,33 @@
+import { Op } from 'sequelize'
+import Models from '../models'
+
+export default async (_, { presence, conversation_id }, { session }) => {
+  const optInStatus = presence ? 'explicit_opt_in' : 'declined'
```

Comment:
you don't wanna make this an enum to protect against mis-typing the string? (since we currently don't do type annotations anywhere)?